loops.js

```diff
@@ -51,3 +67,14 @@
 // >> 9
 // ==========================================
 
+for(let i = 0; i<10; i++) {
+
+            if (i<3)
```

Comment:
Vergeet niet de accolades te gebruiken na de ronde haakjes in de if-statement.
Ook let goed op de spaties tussen de entries en de operatoren.

if (i < 3) {
    console.log(i);
  }